lib/cfg-grammar.y

```diff
@@ -1353,7 +1353,7 @@ msg_format_option
           }
         | KW_SDATA_PREFIX '(' string ')'
           {
-            msg_format_options_set_sdata_prefix(last_msg_format_options, $3);
+            CHECK_ERROR(msg_format_options_set_sdata_prefix(last_msg_format_options, $3), @3, "Prefix is too long \"%s\"", $3);
```

Comment:
[optional]

This can possibly break some old configs. I don't think anyone would run into this, so IMO we can break, but we should point this out in the news file.